modules/insights-operator-new-pull-secret-enable.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * support/insights-operator/enabling-insights-operator.adoc
+
+[id="insights-operator-new-pull-secret-enable_{context}"]
+= Modifying your global cluster pull secret to enable remote health reporting
+
+You can modify your existing global cluster pull secret to enable remote health reporting. If you have previously disabled remote health monitoring, you must first download a new pull secret with your `cloud.openshift.com` access token from Red Hat OpenShift Cluster Manager. 
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to Red Hat OpenShift Cluster Manager.
+
+.Procedure
+
+. Navigate to link:https://console.redhat.com/openshift/downloads[https://console.redhat.com/openshift/downloads]
+. From *Tokens* -> *Pull Secret*, click *Download*. 
++
+The file `pull-secret.txt` containing your `cloud.openshift.com` access token in JSON format downloads:
++
+[source,json,subs="+quotes"]
+----
+{    
+  "auths": {        
+    "cloud.openshift.com": {            
+      "auth": "_<your_token>_",
+      "email": "_<email_address>_"
+    }
+}
+----
+
+. Download the global cluster pull secret to your local file system.
++
+[source,terminal]
+----
+$ oc get secret/pull-secret -n openshift-config --template='{{index .data ".dockerconfigjson" | base64decode}}' > pull-secret
+----
+. Make a backup copy of your pull secret. 
++
+[source,terminal]
+----
+$ cp pull-secret pull-secret-backup
+----
+. Open the `pull-secret` file in a text editor.
+. Append the `cloud.openshift.com` JSON entry from `pull-secret.txt` into `auths`.
+. Save the file.
+. Update the secret in your cluster.
++
+[source,terminal]
+----
+oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=pull-secret
+----
+
+It may take several minutes for the secret to update and your clusters to begin reporting. 
+
+.Verification 
+
+* A green check next to *Insights* in the *Status* tile on the {product-title} Web Console Overview page with the number of issues found. 
```

Comment:
nit: It doesn't have to be green right :) The important point here is that it's not disabled (or unavailable) any more.